docker-compose/metrics/scripts/create-workload-registration-entry.sh

```diff
@@ -48,6 +48,7 @@ docker-compose -f "${PARENT_DIR}"/docker-compose.yaml exec -T spire-server \
     -parentID "spiffe://example.org/spire/agent/x509pop/$(fingerprint "${PARENT_DIR}"/spire/agent/agent.crt.pem)" \
     -spiffeID "spiffe://example.org/workload-A" \
     -selector "unix:uid:1001" \
-    -ttl 120
+    -x509SVIDTTL 120 \
+    -jwtSVIDTTL 120
```

Comment:
does this tutorial use JWT-SVIDs? If not, setting the TTL might be confusing.

Reply:
No, I can remove it.